test/testenv/testenv.go

```diff
@@ -264,20 +265,23 @@ func (testenv *TestEnv) setup() error {
 		return err
 	}
 
-	err = testenv.createRole()
-	if err != nil {
-		return err
-	}
+	/*
+		err = testenv.createRole()
+		if err != nil {
+			return err
+		}
 
-	err = testenv.createRoleBinding()
-	if err != nil {
-		return err
-	}
+		err = testenv.createRoleBinding()
+		if err != nil {
+			return err
+		}
 
-	err = testenv.createOperator()
-	if err != nil {
-		return err
-	}
+
+		err = testenv.createOperator()
+			if err != nil {
+				return err
+			}
+	*/
```

Comment:
nit: remove?

Reply:
need to work with param on this, will wait for him